azure-pipelines.yml

```diff
@@ -87,47 +87,32 @@ stages:
         sudo service redis-server start
 
         # LIBYANG
-        sudo dpkg -i ../target/debs/buster/libyang*1.0.73*.deb
+        sudo dpkg -i ../target/debs/bullseye/libyang*1.0.73*.deb
       displayName: "Install dependency"
 
     - script: |
         # LIBSWSSCOMMON
         sudo apt-get -y purge libhiredis-dev libnl-3-dev libnl-route-3-dev
-        sudo dpkg -i ../target/debs/buster/libnl-3-200_*.deb
-        sudo dpkg -i ../target/debs/buster/libnl-genl-3-200_*.deb
-        sudo dpkg -i ../target/debs/buster/libnl-route-3-200_*.deb
-        sudo dpkg -i ../target/debs/buster/libnl-nf-3-200_*.deb
-        sudo dpkg -i ../target/debs/buster/libhiredis0.14_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libnl-3-200_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libnl-genl-3-200_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libnl-route-3-200_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libnl-nf-3-200_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libhiredis0.14_*.deb
+        sudo dpkg -i ../target/debs/bullseye/libhiredis-dev_*.deb
```

Comment:
>-dev_ [](http://example.com/codeflow?start=55&length=5)

Do you need all the dev packages?

Reply:
yes, build failed without these dev packages.